lib/platform/bitbucket-server/bb-got-wrapper.ts

```diff
@@ -28,7 +28,11 @@ export const api: GotApi = {} as any;
 
 for (const x of helpers) {
   (api as any)[x] = (url: string, opts: any): Promise<GotResponse> =>
-    get(url, { ...opts, method: x.toUpperCase() });
+    get(url, {
+      ...opts,
+      method: x.toUpperCase(),
+      retry: { errorCodes: 'ECONNRESET' },
```

Comment:
Can you contribute this as a separate PR?

Reply:
Yes, I've removed this change from the current PR and I'll raise this later